### PR TITLE
Feat:"added Product Listing Page"

### PR DESCRIPTION
**Pull request by sadathshariff**
Added Filter and Product Listing page

https://user-images.githubusercontent.com/51914072/153820279-4fa07b7f-d94c-42ca-bf0e-60339da47210.mov

### CODE REVIEW

File: CSS/CommonStyles/card.css [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-  height: 30rem;
-  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.7);
+  height: max-content;
   margin: 1rem;
+  background-color: var(--off-white);
 }
 
 .card-img {
-  height: 19rem;
   object-fit: contain;
```
Comment (therajatg) [inline CSS/CommonStyles/card.css:9]: Why not "object-fit: cover". It will cover all the space in a container whatever the container's dimension.

Comment (sadathshariff) [reply CSS/CommonStyles/card.css:9]: Fixed it

**Review by therajatg — APPROVED** [review]
looks good!

**Review by kjais1720 — APPROVED** [review]
Looks good overall, good use of utility classes and proper naming convention

**Pull request merged**